docs/mkdocs.yml

```diff
@@ -867,6 +867,17 @@ extra:
   analytics:
     provider: google
     property: UA-110480857-1
+  consent:
+    actions:
+      - accept
+      - reject
+      - manage
```

Comment:
Maybe we could remove the reject button. If they don't want to consent they can do it through the manage button.

What also could be nice is to have the manage button first and then the accept button.